course_info/templates/course_info/partials/details.html

```diff
@@ -261,7 +261,6 @@ <h3 class="breadcrumbs">
                       <a href="{{site.course_site_url}}" target="_blank">{{site.course_site_url}}</a>
                     </div>
                     <div class="col-sm-3">
-                      <!-- todo: switch $index to site_id when available -->
                       <button type="button" class="btn btn-default pull-right"
                               ng-click="dc.dissociateSite($index)">
```

Comment:
Index could change though while something is querying the backend, I would think -- for instance, we might delete site 1 and site 2, and if site 1 is removed first, the index to site 2 might no longer be valid/reliable. So that's why I think a `site_map_id`-keyed object would be better. That's why the comment was there, sorry for not being clearer initially!